src/Pages/WellieMain/Components/PriceCard.js

```diff
@@ -0,0 +1,178 @@
+            정기구독
+          </strong>
+        </p>
+        <p>
+          <span>월 </span>
+          <span>{props.monthPrice}</span>
+          <br />
+          <span>연 </span>
+          <span>{props.yearPrice} </span>
+          <strike>{props.originalPrice}</strike>
+        </p>
+      </div>
+      <div className="body">
+        <dl>
+          <dt>
+            <span>
+              <FaCheck />
+            </span>
+            전자책 10만 권 무제한
+          </dt>
+          <dt>
+            <span>
+              <FaCheck />
+            </span>
+            밀리 단독 콘텐츠
+            <span> 오디오북, 챗북</span>
+          </dt>
+          <dt>
+            <span>
+              <FaCheck />
+            </span>
+            <span>첫 달 무료</span>
+          </dt>
+          <dd>{props.description1}</dd>
+          <dd>유료 전환시, 미사용 여부가 확인되면 7일 내 환불 100%</dd>
+        </dl>
+      </div>
+      <div className="buttons">
+        <PriceCardBtn month>월 정기구독</PriceCardBtn>
+        <PriceCardBtn>연 정기구독</PriceCardBtn>
+      </div>
+    </EachPriceCard>
+  );
+}
+
+const Card = styled.article`
+  display: block;
+  border-radius: 10px;
+`;
+
+const EachPriceCard = styled(Card)`
+  position: relative;
+  background-color: white;
+  width: 350px;
+  height: 784px;
+  margin: 2px;
+  padding-bottom: 16px;
+  color: rgb(52, 52, 52);
+
+  .header {
+    height: 424px;
+    padding: 16px 8px 16px 16px;
+    background-color: rgb(249, 235, 96);
+    border-radius: 10px 10px 0 0;
+
+    p:first-child {
+      line-height: 30px;
+      font-size: 20px;
+      margin-bottom: 12px;
+
+      span {
+        font-weight: 600;
+      }
+    }
+    p:nth-child(3) {
+      line-height: 50px;
+      font-size: 40px;
+      margin-bottom: 12px;
+      font-weight: 600;
+    }
+    p:last-child {
+      padding-top: 30px;
+      line-height: 40px;
+
+      span:nth-child(1),
+      span:nth-child(4) {
+        font-size: 24px;
+      }
+      span:nth-child(2),
+      span:nth-child(5) {
+        font-size: 32px;
+        font-weight: 600;
+      }
+      strike {
+        color: rgba(52, 52, 52, 0.5);
+        font-size: 24px;
+      }
+    }
+  }
+
+  .body {
+    dl {
+      margin: 16px 12px 16px 0;
+      line-height: 20px;
+      dt {
+        padding: 2px 0 2px 16px;
+        font-weight: bold;
+        font-size: 14px;
+        /* &::before {
+        } */
+        span {
+          padding-right: 10px;
+          opacity: 0.5;
+        }
+        /* &:last-child {
+          color: red;
+        } */
+      }
+      dt:nth-child(3) {
+        span:last-child {
+          color: red;
+          opacity: 1;
+        }
+      }
+      dd {
+        padding-left: 40px;
+        opacity: 0.6;
+        font-size: 12px;
+      }
+    }
+  }
+  .buttons {
+    position: absolute;
+    width: 83%;
+    margin: 0 8.5%;
+    bottom: 0;
+  }
+`;
+
+const PriceCardBtn = styled.button`
+  background-color: ${(props) =>
+    props.month ? "rgb(249, 235, 96)" : "rgb(243, 191, 6)"};
```

Comment:
```suggestion
  background-color: ${({ month }) => month ? "rgb(249, 235, 96)" : "rgb(243, 191, 6)"};
```
함수 파라미터 자리에서 직접 구조분해할당 할수 있다는 사실도 언급해둡니다!

Reply:
오!!! 더 간결해지네요 ㅎㅎ
그런데 이전에는 `{ month } = props` 이런식으로 equal 부호를 써서 직접 할당을 해줬는데, 
이 경우에는 그냥 `{ month }` 이런식으로 간결하게 사용하는 게 신기합니다.

props와 직접적인 연관이 있을 경우에는 항상 이처럼 간결하게 구조분해할당을 할 수 있는 건가요?

Reply:
@JeanBaek 꼭 props가 아니더라도 들어오는 값이 객체, 배열 형태라면 모두 가능합니다!